openml/_api_calls.py

```diff
@@ -68,6 +72,45 @@ def _perform_api_call(call, request_method, data=None, file_elements=None):
     return response.text
 
 
+def _download_minio_file(
+    source: str, destination: Union[str, pathlib.Path], exists_ok: bool = True,
```

Comment:
Not related to the content, but how does `black` accept this? Shouldn't it split this up into three lines?

Reply:
If I remember correctly this was actually formatted by `black` because it fits on one line :)